go.mod

```diff
@@ -150,6 +150,7 @@ require (
 	github.com/jbenet/go-context v0.0.0-20150711004518-d14ea06fba99 // indirect
 	github.com/jmespath/go-jmespath v0.4.0 // indirect
 	github.com/jmoiron/sqlx v1.3.5 // indirect
+	github.com/joho/godotenv v1.5.1 // indirect
```

Comment:
How did this end up here?

Reply:
This was a leftover from helper code I had added to a `suite_test.go` to read my `.envrc` directly and hence be able to debug test cases from within VS Code. Happy to share that at some point if someone is interested, or to discover better ways of using env vars there :)